### ISSUE TITLE
cd12 directory is missing

### ISSUE THREAD

**Issue by ckoryom** [foone/3DMMForever#16]
On the fonts.md it's mentioned that we need to place our fonts into ./cd12, unfortunately it seems like the directory has been removed.

Is this on purpose?

**Comment by bjrkk** [comment]
CD12 was merged as part of pull request #4. Placing the fonts is not necessary in order to build 3DMM itself, and is a mere remnant from the original repository.

**Comment by foone** [comment]
Yeah. cd12 was removed for cleanup. I'll make a fix to fonts.md to make this clearer. 
We don't really need to place fonts anywhere as part of the new install process, so maybe just removing fonts.md would make more sense?

**Comment by ckoryom** [comment]
I actually got confused with the README, thought we needed to find the fonts and add them manually. Maybe just adding that extra part "Placing the fonts is not necessary anymore" but keeping fonts.md for historical purposes?
